etl/steps/data/garden/emissions/2024-04-08/national_contributions.meta.yml

```diff
@@ -214,16 +278,22 @@ tables:
         description_key:
           - This temperature change measures each country's contribution to global mean surface temperature (GMST) rise from its cumulative emissions of carbon dioxide, methane and nitrous oxide.
           - The warming effects of each gas are calculated based on cumulative CO₂-equivalent emissions using the Global Warming Potential (GWP*) approach.
```

Comment:
You could maybe expand on what Global Warming Potential (GWP*) approach is

Reply:
We have a DoD for GWP. I suppose it could be better to add a sentence or a paragraph about GWP* there.